conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,20 @@
+# patch 2
+# assets and audits
+
+# --- !Ups
+
+ALTER TABLE `assets` ADD COLUMN `hidden` tinyint(1) NOT NULL DEFAULT 0;
+
+CREATE TABLE `folders` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
```

Comment:
why have an id? Isn't the key an id enough? If you're looking to shrink the index, can we use a truncated md5 instead of autoincrement?